app/views/artefacts/form/_tags.html.erb

```diff
@@ -1,57 +1,60 @@
-<div class="well">
-  <%= f.inputs "Tags" do %>
-    <% if f.object.allow_section_tag_changes? %>
-      <div class="section-tags fieldset-section">
-        <label for="artefact[sections][]" class="section-label">Sections</label>
-
-        <p>The first section will be the primary section the content lives in. This will form the content breadcrumb.</p>
-        <div class="add-vertical-margins nested-item-group">
-          <% if f.object.section_ids.length > 0 %>
-            <% f.object.section_ids.each do | section_id | %>
-              <%= render :partial => 'artefact_section', :locals => {:section_id => section_id, :tag_collection => tag_collection} %>
+<%- if f.object.taggable? %>
+
+  <div class="well">
+    <%= f.inputs "Tags" do %>
+      <% if f.object.allow_section_tag_changes? %>
+        <div class="section-tags fieldset-section">
+          <label for="artefact[sections][]" class="section-label">Sections</label>
+
+          <p>The first section will be the primary section the content lives in. This will form the content breadcrumb.</p>
+          <div class="add-vertical-margins nested-item-group">
+            <% if f.object.section_ids.length > 0 %>
+              <% f.object.section_ids.each do | section_id | %>
+                <%= render :partial => 'artefact_section', :locals => {:section_id => section_id, :tag_collection => tag_collection} %>
```

Comment:
Might be worth giving this file a sweep to replace the hash rockets with 1.9 style keys...